swap/swap.go

```diff
@@ -178,6 +179,13 @@ func New(dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, params *Par
 		return nil, err
 	}
 	swapLog.Info("Using backend network ID", "ID", chainID.Uint64())
+
+	var paidOut uint64
+	if paidOut, err = loadPaidout(stateStore); err != nil {
+		return nil, err
+	}
+	swapLog.Info("Total paidOut in Wei", "paidOut", paidOut)
```

Comment:
no need to use camel case in text that is logged

Reply:
Ok!

Reply:
can you please explain why this was removed?

Reply:
nevermind, just saw the comment below